src/Containers/Piano.jsx

```diff
@@ -60,32 +82,22 @@ export default class Piano extends React.Component {
   }
 
   selectNaturals() {
-    const stateChange = {
-      "C": true,
-      "D": true,
-      "E": true,
-      "F": true,
-      "G": true,
-      "A": true,
-      "B": true,
-    };
+    const stateChange = {};
+    naturals.forEach((key) => {
```

Comment:
Maybe use https://lodash.com/docs/4.17.15#keyBy and https://lodash.com/docs/4.17.15#mapValues in a chain to generate it?